### PR TITLE
Lumbricus example

### PR DESCRIPTION
**Pull request by luitjansl**
added python scripts generating modflow, metaswap and dflowfm models. 
In order to limit the size of the repository, not everything was added to the repository. In order to get the rest, copy N:\Deltabox\Postbox\Slooten, Luit Jan\voor_lumbricus\dflowfm_dll.7z to your machine and unpack it in imod_coupler\examples\modflow_msw_dfm_simulation\dflowfm_dll

### CODE REVIEW

**Review by Hofer-Julian — COMMENTED** [review]
I had a quick look and it looks good so far.
Two small comments, will look in more detail on Wednesday.

File: .gitignore [hunk 1]
```diff
@@ -133,3 +133,103 @@ dmypy.json
 
 # Tests
 tests/temp
+/examples/modflow_msw_dfm_simulation/dflowfm_dll/cilkrts20.dll
```
[the remark this answers is not in the document]
Comment (luitjansl) [reply .gitignore:136]: done

File: examples/modflow_msw_dfm_simulation/create_dflowfm_model.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+    os.mkdir(workdir)
+    os.chdir(workdir)
+
+    # Create new model object
+    fm_model = FMModel()
+    fm_model.filepath = Path(workdir + os.sep + f"{modelname}.mdu")
+
+    network = fm_model.geometry.netfile.network
+    # This will can only be used as soon as https://github.com/Deltares/HYDROLIB-core/issues/290 is solved
```
[the remark this answers is not in the document]
Comment (luitjansl) [reply examples/modflow_msw_dfm_simulation/create_dflowfm_model.py:40]: done

**Comment by luitjansl** [comment]
closed but not merged.

**Pull request closed without merging**